examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,130 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import glide.api.GlideClient;
+import glide.api.logging.Logger;
+import glide.api.models.configuration.GlideClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.TimeoutException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a GlideClient instance. This function initializes a
+     * GlideClient with the provided list of nodes.
+     *
+     * @return A <code>GlideClient</code> connected to the discovered nodes.
```

Comment:
nodes aren't getting discovered in standalone, please fix according to https://github.com/valkey-io/valkey-glide/blob/main/examples/python/standalone_example.py#L33